mne/epochs.py

```diff
@@ -1690,6 +1721,24 @@ def save(self, fname, split_size='2GB', verbose=True):
                                       '_epo.fif', '_epo.fif.gz'))
         split_size = _get_split_size(split_size)
 
+        if self.load_data():
+            if np.iscomplexobj(self._data):
+                warn('Saving epoched file with complex data. Loading with '
+                     'command-line MNE tools will not work.')
+
```

Comment:
Running the test using pytest these line produces a `RuntimeWarning `classifying the test as failed. Commenting this warning, the test is passed. I am not sure whether that's a problem

Reply:
Enclose the failing call in the test with:
```
with pytest.warns(RuntimeWarning, match='complex data'):
```